pyk/src/pyk/tests/mock_kprint.py

```diff
@@ -0,0 +1,13 @@
+from typing import cast
+
+from ..kast import KAst
+from ..ktool.kprint import KPrint, prettyPrintKast
+
+
+class MockKPrint:
```

Comment:
Did you consider implementing this with inheritance?

Also, if this class is used in only a single test, let's make it local to that module.

Reply:
Its used by both the `KCFG` pretty print and `Subst` now.
Oh! Didn't realize we could just not call the constructor. I'm more familiar with mocking in C++.